pages/common/cmake.md

```diff
@@ -1,13 +1,21 @@
 # cmake
 
-> Cross-platform build system generator.
-> It generates Makefiles, Visual Studio projects or others, depending on the target system.
+> Cross-platform build, testing and packaging automation system.
+> CMake uses own shell-like syntax and generates recipes for several build systems.
 > More information: <https://cmake.org/cmake/help/latest/manual/cmake.1.html>.
 
-- Generate a Makefile and use it to compile a project in the same directory as the source:
+- Generate a build recipe in the current directory with CMakeLists.txt from a project directory:
 
-`cmake && make`
+`cmake {{path/to/project_dir/}}`
 
-- Generate a Makefile and use it to compile a project in a separate "build" directory (out-of-source build):
+- Build with the generated recipe in build_dir (artifacts go to build_dir):
 
-`cmake -H. -B {{build}} && make -C {{build}}`
+`cmake --build {{path/to/build_dir/}}`
+
+- Install the project:
```

Comment:
Could you specify where the files get installed, something like `Install the built artifacts to the system's default folders`?

Reply:
It hardly depends on a project setup and build system imho.

Reply:
I just thinking that "Install the project" probably wouldn't mean much to a new user.

Reply:
I updated example to be more verbose.